### PR TITLE
Bugfix: Shortcode != short

### PR DESCRIPTION
**Pull request by devmaxde**
The wrong param is used

### CODE REVIEW

**Comment by mibkvr** [comment]
https://dashflo.net/docs/api/pterodactyl/v1/#req_a52b4fb62ef7473db08747e8723ab425

**Comment by iamkubi** [comment]
The docs aren't always correct and links to specific functions have sadly never worked.  Have you tested this?

**Comment by mibkvr** [comment]
Yeah. I saw that, because my program did some strange bugs

**Comment by mibkvr** [comment]
I can provide you a screenshot if you want

File: pydactyl/api/locations.py [hunk 1]
```diff
@@ -31,33 +31,33 @@ def get_location_info(self, location_id, includes=None, params=None):
             includes=includes, params=params)
         return response
 
-    def create_location(self, shortcode, description):
```
Comment (iamkubi) [inline pydactyl/api/locations.py:34]: Don't change the name of the parameters, this isn't necessary and it breaks anyone who is specifying these as named parameters.

**Comment by iamkubi** [comment]
> I can provide you a screenshot if you want

No worries, just checking if this was purely based on what the docs said or based on an actual bug.  Totally believe it's not working and we'll get it fixed.

**Comment by mibkvr** [comment]
Update on that?

**Comment by iamkubi** [comment]
> Update on that?

You never responded / fixed the review comment.

**Comment by mibkvr** [comment]
Fixed it by now :D

**Pull request closed without merging**